h/services/search_index/service.py

```diff
@@ -2,7 +2,7 @@
 
 from h import storage
 from h.presenters import AnnotationSearchIndexPresenter
-from h.tasks.indexer import add_annotation, delete_annotation
+from h.tasks import indexer
```

Comment:
Importing `add_annotation`, `delete_annotation` and (now) `add_annotations_between_times` as top-level names was confusing because `service.py` also contains its own methods named `SearchIndexService.add_annotation()`, `add_annotation_by_id()`, `delete_annotation` and `add_annotations_between_times`. I think the code is clearer if, whenever it's accessing the `tasks.indexer` stuff, it's always `indexer.foo()`. This also enables the tests to have a patch fixture for this import and just name the patch fixture `indexer` (our usual naming style for patch fixture). You couldn't have patch fixtures named `add_annotation` and `delete_annotation`, or at least it'd be really confusing to, because one test class in `service_test.py` already contains fixtures with those names (that patch `SearchIndexService.add_annotation` and `delete_annotation`).